CHANGELOG.rst

```diff
@@ -4,6 +4,33 @@
 Synapse Changelog
 *****************
 
+v2.11.0 - 2020-12-30
+====================
+
+Features and Enhancements
+-------------------------
+- Add a ``onload`` paramter to the ``stormpkg`` definition. This represents
+  a Storm query which is executed every time the ``stormpkg`` is loaded in
+  a Cortex.
+  (`#1971 <https://github.com/vertexproject/synapse/pull/1971>`_)
+- Add the ability, in Storm, to unset variables, remove items from
+  dictionaries, and remove items from lists. This is done via assigning
+  ``$lib.unset`` to the value to be removed.
+  (`#1970 <https://github.com/vertexproject/synapse/pull/1970>`_)
+- Add support for SOCKS proxy support to the Axon and Cortex, using the
```

Comment:
Suggest `Add support for SOCKS proxy support for outgoing connection from an Axon and Cortex```